core/jazz_acl/components/validation.js

```diff
@@ -16,23 +16,47 @@
 
 const errorHandlerModule = require("./error-handler.js")();
 
-async function validateInput(userInput) {
+function validateBasicInput(userInput) {
 
 		if (!userInput || !userInput.method) {
 			throw (errorHandlerModule.throwInputValidationError("Invalid or missing arguments"));
     }
 
-    if (!userInput.resourcePath) {
+    if (!userInput.path) {
 			throw (errorHandlerModule.throwInputValidationError("Missing the resource path"));
```

Comment:
`path` is usually set by API Gateway. If it's missing, it's more of a system integration error (`InternalServerError`)

Reply:
This logic will take care of empty/missing values as well.
i will update the one above(for method to remove userInput.